interfaces/src/db.ts

```diff
@@ -1,5 +1,5 @@
 import { Operation } from 'fast-json-patch';
-export type Comparable = string | number | Date;
+export type Comparable = string | number;
```

Comment:
The fix is removal of Date from types?

Reply:
Yes... it was never officially supported and will have to be revisited.